test/functorch/test_eager_transforms.py

```diff
@@ -1323,7 +1323,7 @@ def vmap(info, in_dims, input):
     def test_in_dims_multiple_inputs(self, device):
         class Id(torch.autograd.Function):
             @staticmethod
-            def forward(input):
+            def forward(x, y):
```

Comment:
Ah this is technically bc-breaking lol, since I guess there are cases like vmap where we previously didn't even execute forward.